src/components/autosuggest-text/autosuggest-text-item.tsx

```diff
@@ -30,6 +30,7 @@ const AutosuggestTextItem: React.FC<AutosuggestTextItemProps> = ({
         className={classes.textSuggestion}
         key={generateItemId(item)}
         {...getItemProps({ item, index })}
+        data-cy="autosuggest-text-item"
```

Comment:
Normally when it is a component that needs a data-cy attribute on the outer/first level we make it possible to overwrite via a prop. Like [this](https://github.com/reload/dpl-react/blob/cdfa637892e619a032f5a924e92fd7f5964cf328/src/components/availability-label/availability-label.tsx#L32) and [this](https://github.com/reload/dpl-react/blob/cdfa637892e619a032f5a924e92fd7f5964cf328/src/components/availability-label/availability-label.tsx#L93-L94). In that way it canvary according to the context it is used in.